Aftonbladet.js

```diff
@@ -0,0 +1,209 @@
+{
+	"translatorID": "235c5c47-0c04-4d77-9afe-e22265a017a9",
+	"label": "Aftonbladet",
+	"creator": "Jonatan Svensson Glad",
+	"target": "^https?://ww*\.aftonbladet\.se",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2016-08-10 12:53:00"
+}
+
+/*
+	Aftonbladet Translator - Parses Atonbladet articles and creates Zotero-based metadata.
+	Copyright (C) 2016 Jonatan Svensson Glad
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU General Public License for more details.
+	You should have received a copy of the GNU General Public License
+	along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+*/
+
+
+/* Zotero API */
+
+function detectWeb(doc, url) {
+    //TODO: adjust the logic here
+    if (url.indexOf('/article')>-1) {
+        return "newspaperArticle";
+    } else if (getSearchResults(doc, true)) {
+        return "multiple";
+    }
+}
+
+
+function getSearchResults(doc, checkOnly) {
+    var items = {};
+    var found = false;
+    //TODO: adjust the xpath
+    var rows = ZU.xpath(doc, '//a[contains(@href, "/article")]');
+    for (var i=0; i<rows.length; i++) {
+        //TODO: check and maybe adjust
+        var href = rows[i].href;
+        //TODO: check and maybe adjust
+        var title = ZU.trimInternal(rows[i].textContent);
```

Comment:
You can try out here
```
var title = ZU.xpathText(rows[i], './/h2');
```
instead, which should result in nicer titles. However, I am not sure if captured all relevant articles on that page.